setup.py

```diff
@@ -15,5 +15,5 @@
     url="http://github.com/joshmarshall/testnado/",
     license="http://www.apache.org/licenses/LICENSE-2.0",
     packages=find_packages(exclude=["tests", "dist"]),
-    install_requires=["tornado", "selenium", "mock"]
+    install_requires=["tornado>=4", "mock"]
```

Comment:
I don't know what it would take to do this, but it'd be nice if `mock` was only required when installing into a Python 2 environment.

I see you've removed Python 2 from the Travis builds. Does that mean you've dropped Python 2 support and `mock` can be dropped from the requirements, entirely?